clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/MemoryPersistenceDAO.scala

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.clio.server.dataaccess
+
+import org.broadinstitute.clio.server.dataaccess.elasticsearch.{
+  ClioDocument,
+  ElasticsearchIndex
+}
+
+import com.sksamuel.elastic4s.Indexable
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class MemoryPersistenceDAO extends MockPersistenceDAO {
```

Comment:
Do we need both `Jimfs` and `Memory` DAOs for tests? Seems like Jimfs could be used for the memory case as well.